src/libarena/lib.rs

```diff
@@ -297,7 +290,8 @@ impl<'longer_than_self> Arena<'longer_than_self> {
 
         unsafe {
             let buf = head.as_ptr();
-            (buf.offset(tydesc_start as isize), buf.offset(start as isize))
```

Comment:
There's a heuristic where we cut off into vertical formatting before we hit the absolute limit. We probably shouldn't do it for tuples though (only for function calls).

https://github.com/nrc/rustfmt/issues/450